addons/core/functions/fnc_haveProgrammator.sqf

```diff
@@ -20,4 +20,10 @@
 
 if (isNil {TFAR_currentUnit} || {isNull (TFAR_currentUnit)}) exitWith {false};
 
-"TFAR_microdagr" in (assignedItems TFAR_currentUnit);
+_result = "TFAR_microdagr" in (assignedItems TFAR_currentUnit);
+
+if (player != TFAR_currentUnit && {TFAR_remoteRadio}) then {
+    _result = "TFAR_microdagr" in (assignedItems player);
```

Comment:
Same as above. We scan all assigned Items.. Only to find out we actually didn't need to do that and discard the previous result.